src/main/webapp/i18n/de/userSettings.json

```
@@ -27,6 +28,20 @@
                 "sshKeyDisplayedInformation": "Das ist dein aktuell konfigurierter SSH-Schlüssel:",
                 "key": "SSH Schlüssel"
             },
+            "vcsAccessTokensSettingsPage": {
+                "infoText": "Du kannst ein persönliches Zugriffstoken generieren, um mit der Artemis Local Version Control System zu interagieren. Verwende es um dich über HTTP bei Git zu authentifizieren.",
```

```suggestion
                "infoText": "Du kannst ein persönliches Zugriffstoken generieren, um mit dem Artemis Local Version Control System zu interagieren. Verwende es um dich über HTTP bei Git zu authentifizieren.",
```
Do we have a better German name than `Artemis Local Version Control System`?